iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/PublicNetworkAccessCheck.java

```diff
@@ -132,6 +132,11 @@ public class PublicNetworkAccessCheck extends AbstractArmResourceCheck {
     "Microsoft.DocumentDB/mongoClusters/firewallRules",
     "Microsoft.Sql/servers/firewallRules",
     "Microsoft.Synapse/workspaces/firewallRules");
+
+  private static final List<String> PUBLIC_NETWORK_ACCESS_SIMPLIFIED_IN_SITE_CONFIG = List.of(
```

Comment:
Why is this also `SIMPLIFIED`? Shouldn't this only be `PUBLIC_NETWORK_ACCESS_SIMPLIFIED_IN_SITE_CONFIG `? Same for all related methods and test files

Reply:
The `publicNetworkAccess` property in some cases can have the following sensitive values: `"Enabled", "EnabledForSessionHostsOnly", or "EnabledForClientsOnly" ` and in other cases only `Enabled` is sensitive. I call the second case `SIMPLIFIED`. If you have a better name for it, I'm happy to refactor it.

Reply:
Ah ok, I thought it had something to do with how deep the `enabled` is inside the properties.